saleor/graphql/core/types/common.py

```diff
@@ -50,3 +51,31 @@ class Weight(graphene.ObjectType):
 
     class Meta:
         description = 'Represents weight value in a specific weight unit.'
+
+
+class Image(graphene.ObjectType):
+    url = graphene.String(
+        required=True,
+        description='The URL of the image.')
+    alt = graphene.String(description='Alt text for an image.')
+
+    class Meta:
+        description = 'Represents an image.'
+
+    @staticmethod
+    def get_adjusted(image, alt, size, rendition_key_set, info):
+        """Return Image adjusted with given size."""
+
+        if size:
+            url = get_thumbnail(
+                image_file=image,
+                size=size,
+                method='thumbnail',
+                rendition_key_set=rendition_key_set,
+            )
+        else:
+            url = image.url
+
+        url = info.context.build_absolute_uri(url)
+
```

Comment:
I guess we don't need those blank lines - there is not much logic in this function to separate for better readability.